Microsoft.Toolkit.Win32.UI.XamlHost/IDesktopWindowXamlSourceNative.cs

```diff
@@ -4,23 +4,23 @@
 
 using System;
 using System.Runtime.InteropServices;
-using windows = Windows;
+using WUX = Windows.UI.Xaml;
 
 namespace Microsoft.Toolkit.Win32.UI.XamlHost
 {
     /// <summary>
     /// Enables access to native methods on DesktopWindowXamlSourceNative
     /// </summary>
     /// <remarks>
-    /// Includes the method used to set the window handle of the <see cref="windows.UI.Xaml.Hosting.DesktopWindowXamlSource" /> instance.
+    /// Includes the method used to set the window handle of the <see cref="WUX.Hosting.DesktopWindowXamlSource" /> instance.
     /// </remarks>
     [ComImport]
     [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
     [Guid("3cbcf1bf-2f76-4e9c-96ab-e84b37972554")]
-    public interface IDesktopWindowXamlSourceNative
+    partial interface IDesktopWindowXamlSourceNative
```

Comment:
Can IDWXS2 (or the most recent) always be used and this file removed?

Reply:
No, we still need to support RS4 clients.

---
In reply to: [261452136](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r261452136) [](ancestors = 261452136)